main/options/options.cc

```diff
@@ -353,6 +353,8 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
         cxxopts::value<vector<string>>(), "string");
     options.add_options("advanced")("no-error-count", "Do not print the error count summary line");
     options.add_options("advanced")("autogen-version", "Autogen version to output", cxxopts::value<int>());
+    options.add_options("advanced")("report-ambiguous-defs", "Enable ambiguous definition error enforcement",
```

Comment:
nit: what do you think about lengthening `defs` to `definitions` or maybe `constant-definitions`?

e.g., I usually associate "defs" with the `def` keyword for methods in Ruby, but iiuc this feature is entirely related to constants.